akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala

```diff
@@ -173,12 +183,13 @@ private[cluster] final class ClusterHeartbeatSender extends Actor with ActorLogg
  * State of [[ClusterHeartbeatSender]]. Encapsulated to facilitate unit testing.
  * It is immutable, but it updates the failureDetector.
  */
+@InternalApi
 private[cluster] final case class ClusterHeartbeatSenderState(
   ring:                       HeartbeatNodeRing,
   oldReceiversNowUnreachable: Set[UniqueAddress],
   failureDetector:            FailureDetectorRegistry[Address]) {
 
-  val activeReceivers: Set[UniqueAddress] = ring.myReceivers union oldReceiversNowUnreachable
+  val activeReceivers: immutable.Set[UniqueAddress] = ring.myReceivers union oldReceiversNowUnreachable
```

Comment:
True, though we seem to use that style even for the set in many places - I'll change to just Set